src/app/core/cartv2.service.ts

```diff
@@ -6,6 +6,7 @@ import { saveAs as importedSaveAs } from 'file-saver';
 import { AuthService } from './auth.service';
 import { BehaviorSubject, throwError } from 'rxjs';
 import { catchError, retry } from 'rxjs/operators';
+import { stringify } from '@angular/compiler/src/util';
```

Comment:
Was this an accidental auto import? I don't see it used here

Reply:
Resolved with remove stringify.